src/tests/query.rs

```diff
@@ -84,6 +84,60 @@ async fn transactions(api: &mut dyn TestApi) -> crate::Result<()> {
     Ok(())
 }
 
+#[test_each_connector(tags("mssql", "postgresql", "mysql"))]
+async fn transactions_with_isolation_works(api: &mut dyn TestApi) -> crate::Result<()> {
```

Comment:
I don't have a great idea how to test that the TX isolation levels are set correctly, this is the best I came up with.